yarn.lock

```diff
@@ -1287,9 +1287,9 @@
     defer-to-connect "^2.0.0"
 
 "@tinymce/eslint-plugin@^1.8.1":
-  version "1.8.2"
-  resolved "https://registry.yarnpkg.com/@tinymce/eslint-plugin/-/eslint-plugin-1.8.2.tgz#5e70f59196f29b10363585f32a0b69d28f663205"
-  integrity sha512-18s/lxY6knDLsW/xICS/qjJSESBHiIJU4nzJkEkTquakENoHjWJonP8dm81OhfjYqyS2h6DRVtyIpoSLZrLVlA==
+  version "1.9.1"
```

Comment:
What are your thoughts on also bumping the version in the package.json from ^1.8.1 to ^1.9.2?

Reply:
Point. I've converted it to use `^1.9.1`.